docs/src/whatsnew/3.2.rst

```diff
@@ -38,7 +38,9 @@ v3.2.1 |build_date| [unreleased]
 
    🐛 **Bugs Fixed**
 
-   #. N/A
+   #. `@dennissergeev`_ changed _crs_distance_differentials() so that it uses the `Globe`
+      attribute from a given CRS instead of creating a new `ccrs.Globe()` object.
+      This allows iris to deal with non-Earth semi-major axes.
```

Comment:
Just something tiny - can your what's new entry mention the issue and PR so it's easy for people to trace them, like the ones here do? https://scitools-iris.readthedocs.io/en/stable/whatsnew/latest.html#bugs-fixed

Reply:
Sure! Just added.